src/Morphir/Visual/ViewValue.elm

```diff
@@ -181,20 +177,16 @@ viewValueByLanguageFeature config value =
                         , smallSpacing config.state.theme |> spacing
                         , onClick (config.handlers.onReferenceClicked fQName False)
                         ]
-                        [ Element.el []
-                            (text
-                                (nameToText localName)
-                            )
-                        ]
+                        [ text (nameToText localName) ]
 
                 Value.Field ( _, _ ) subjectValue fieldName ->
                     let
                         defaultValue =
                             Element.row
-                                [ spacing 1 ]
+                                [ smallPadding config.state.theme |> padding, spacing 1, alignLeft ]
                                 [ viewValue config subjectValue
                                 , el [ Font.bold ] (text ".")
-                                , text (nameToText fieldName)
+                                , text (toCamelCase fieldName)
```

Comment:
Because if we want to display something like `record with long name . field function with long name` it's hard to find the "." so it's hard to interpret which name belongs where in my opinion.